operations/iscsi_sbps/Managing_selective_node_personalization.md

```diff
@@ -0,0 +1,85 @@
+# Managing selective node personalization
+
+Example command(s) to manage selective worker node personalization for iSCSI SBPS:
```

Comment:
The HPE docs style guide says to avoid using `(s)` at the end of words. Just make it plural. If it is really important to highlight that it may be one or it may be more than one, you can say `command or commands`. But in this case, I think just making it plural makes the most sense.
```suggestion
Example commands to manage selective worker node personalization for iSCSI SBPS:
```